library/alloc/src/vec/clone_on_write.rs

```diff
@@ -0,0 +1,34 @@
+use crate::borrow::Cow;
```

Comment:
It might be good to just name this file `cow.rs`. I think the word `Cow` is more recognizable in Rust than `clone_on_write`.